roles/deploy/tasks/deploy.yml

```diff
@@ -95,6 +95,14 @@
     - rake
     - skip_ansible_lint
 
+- name: clear transient data
+  command: bash -lc "bundle exec rake ofn:data:remove_transient_data RAILS_ENV={{ rails_env }}"
+  args:
+    chdir: "{{ build_path }}"
+  tags:
+    - rake
+    - skip_ansible_lint
+
```

Comment:
why don't we then run it once a week instead of coupling it to the deployment? This will make deployments slower and brings a 2nd way to trigger this, while we might not need it.

If what we want instead is to make sure it's run in all instances soon why don't we just do so manually once?

Reply:
I see your point. Slowing down the deployment isn't great.

The intention here is to speed up migrations. The best timing for that is to run it just before the migration.

I often find event based triggers better than time based ones. Imagine you randomly deploy at the same time as the cleaning is run. The database should catch that but you could run into a deadlock depending on the migration.

Reply:
> Imagine you randomly deploy at the same time as the cleaning is run

but that still happens at 4am in the morning or similar. So it's very unlikely.